frontend/src/stimulus/controllers/external-links.controller.ts

```diff
@@ -29,61 +29,84 @@
 import { ApplicationController } from 'stimulus-use';
 import { useMutation } from 'stimulus-use';
 
-const BLANK_LINK_QUERY = 'a[target="_blank"]';
+const EXTERNAL_LINK_QUERY = 'a[href^="http"]';
 const BLANK_LINK_DESCRIPTION_ID = 'open-blank-target-link-description';
```

Comment:
nitpick: Perhaps we can rename the constant for consistency?

```suggestion`
const EXTERNAL_LINK_DESCRIPTION_ID = 'open-external-link-description';
```